SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -94,6 +116,12 @@ class SPATIALGDK_API ASpatialDebugger :
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = Visualization, meta = (ToolTip = "Color used for any server with an unresolved name"))
 	FColor InvalidServerTintColor = FColor::Magenta;
 
+	UPROPERTY(VisibleAnywhere, ReplicatedUsing = OnRep_SetWorkerRegions, Category = Visualization, meta = (ToolTip = "Load-balancing worker region data replicated to clients"))
```

Comment:
Is VisibleAnywhere required?

Reply:
The compiler complains that there needs to be some specifier, I wasn't really sure which was apt. We don't want to be able to edit at all in blueprint - do you have other suggestions?

Reply:
Does it work if you also remove everything after the OnRep function name?

Reply:
```suggestion
	UPROPERTY(ReplicatedUsing = OnRep_SetWorkerRegions)
```